specification/semantic_conventions.md

```diff
@@ -0,0 +1,27 @@
+# Semantic Conventions
+
+This document defines the OpenTelemetry and Splunk specific standard attributes for Splunk distributions of OpenTelemetry.
+
+## OpenTelemetry Resource Attributes
+
+**Description:** Required and recommended OpenTelemetry [resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk).
+
+All Splunk distributions of OpenTelemetry,
+
+- MUST set the following OpenTelemetry resource attributes according to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk):
+  - `telemetry.sdk.name`
+  - `telemetry.sdk.version`
+  - `telemetry.sdk.language`
+
+- SHOULD set the following resource attributes when applicable:
+  - `telemery.auto.version`
+
+
+## Splunk Resource Attributes
+
+**Description:** Set of attributes used to uniquely identify a Splunk distro version in combination with OpenTelemetry's `telemetry.sdk.*` attributes.
+
+
+| Attribute  | Type | Description  | Examples  | Required |
+|---|---|---|---|---|
+| `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
```

Comment:
Do we have a definition what a "Splunk distro" is?
For me it is some convenience library for creating the global tracer. Is my intuition correct? Maybe it would good to make it more clear.

Do we have (or want) any attributes for the instrumentation libraries so that we could identify which are used?

Reply:
> to make it clear by adding a definition here: https://github.com/signalfx/gdi-specification#terms

May be but this PR shouldn't care about that IMO.

> Do we have (or want) any attributes for the instrumentation libraries so that we could identify which are used?

Already exists:

https://github.com/open-telemetry/oteps/blob/main/text/0083-component.md#internal-details
https://github.com/open-telemetry/opentelemetry-specification/blob/d001d5d25ea11a28bd4c4ec2652e2db1a41fc064/specification/trace/sdk_exporters/non-otlp.md#instrumentationlibrary